pkg/resources/model/system.go

```diff
@@ -281,46 +283,49 @@ func (l *LoggingResources) CreateFlowFromCustomResource(flowCr interface{}) (*ty
 	flow := commonFlow.Flow
 	outputs := []types.Output{}
 	var flowType string
-	if commonFlow.Scope != "" {
+	if commonFlow.Scoped {
 		flowType = "flow"
 	} else {
 		flowType = "clusterflow"
 	}
 	var multierr error
+
 FindOutputForAllRefs:
 	for _, outputRef := range commonFlow.OutputRefs {
-		// only namespaced flows should use namespaced outputs
-		if commonFlow.Scope != "" {
```

Comment:
Is the intent here to allow clusterflows accessing namespaced outputs?

Reply:
No, why would you think that?

Reply:
Because the condition has been removed, and I can't see what replaces it. (It's not necessarily a bad idea, just not something I would have expected.)

Reply:
The check has been replaced by [`FlowDispatcher` assigning `ClusterFlow`s' `OutputRefs` to `CommonFlow`s' `ClusterOutputRefs`](https://github.com/banzaicloud/logging-operator/pull/550/files#diff-a9c7f29e4214f03e1e85112070db41bfR211).

In a way, you're right, cluster flows _could_ also reference namespaced outputs (if their CRDs were extended and the `ClusterFlow`→`CommonFlow` conversion logic adjusted accordingly), but only from their own namespace, which is the control namespace — and while cluster flows are still namespaced resources, which is not for long.